test/evp_test.c

```diff
@@ -1817,8 +1817,10 @@ static int encode_test_run(EVP_TEST *t)
             goto err;
 
         EVP_EncodeInit(encode_ctx);
-        EVP_EncodeUpdate(encode_ctx, encode_out, &chunk_len,
-                         expected->input, expected->input_len);
+        if (!TEST_true(EVP_EncodeUpdate(encode_ctx, encode_out, &chunk_len,
+                                        expected->input, expected->input_len))){
+            goto err;
+        }
```

Comment:
Coding style nit: braces not necessary. Personally, I wouldn't complain if you prefer to leave them in place, but then you would need to insert a space before the opening brace in line 1821: `){` -> `) {`